example/ZXingReader.cpp

```diff
@@ -102,7 +102,7 @@ void drawLine(const ImageView& image, PointI a, PointI b)
 	PointF dir = bresenhamDirection(PointF(b - a));
 	for (int i = 0; i < steps; ++i) {
 		auto p = centered(a + i * dir);
-		*((uint32_t*)image.data(p.x, p.y)) = 0xff0000ff;
+		*((uint32_t*)image.data((int)p.x, (int)p.y)) = 0xff0000ff;
```

Comment:
instead of this cast, please replace the line above with `auto p = PointI(centered(a + i * dir));`

Reply:
This and previous suggestions done via force-push https://github.com/nu-book/zxing-cpp/pull/299/commits/6c215b861b014e08529b099270fbf564c0621a51